sycl/include/sycl/ext/oneapi/experimental/group_helpers_sorters.hpp

```diff
@@ -48,31 +56,35 @@ template <typename Group, size_t Extent> class group_with_scratchpad {
 };
 
 // ---- sorters
-template <typename Compare = std::less<>> class default_sorter {
-  Compare comp;
+namespace default_sorters {
+
+template <typename CompareT = std::less<>> class joint_sorter {
+  CompareT comp;
   std::byte *scratch;
   size_t scratch_size;
 
 public:
   template <size_t Extent>
-  default_sorter(sycl::span<std::byte, Extent> scratch_,
-                 Compare comp_ = Compare())
+  joint_sorter(sycl::span<std::byte, Extent> scratch_,
+               CompareT comp_ = CompareT())
       : comp(comp_), scratch(scratch_.data()), scratch_size(scratch_.size()) {}
 
   template <typename Group, typename Ptr>
   void operator()(Group g, Ptr first, Ptr last) {
+    (void)g;
+    (void)first;
+    (void)last;
 #ifdef __SYCL_DEVICE_ONLY__
     using T = typename sycl::detail::GetValueType<Ptr>::type;
     if (scratch_size >= memory_required<T>(Group::fence_scope, last - first))
       sycl::detail::merge_sort(g, first, last - first, comp, scratch);
       // TODO: it's better to add else branch
 #else
-    (void)g;
-    (void)first;
-    (void)last;
+
     throw sycl::exception(
         std::error_code(PI_ERROR_INVALID_DEVICE, sycl::sycl_category()),
-        "default_sorter constructor is not supported on host device.");
+        "default_sorters::joint_sorter constructor is not supported on host "
+        "device.");
 #endif
   }
 
```

Comment:
Now the code at Line 91 - Line 110 is a part of `group_sorter`. Please, remove it here

Reply:
Correct! Removed. Thanks!